sdk/include/aditof/log_cout.h

```diff
@@ -0,0 +1,7 @@
+#include <iostream>
+
+#define INFO "INFO "
+#define ERROR "ERROR "
+#define WARNING "WARNING "
+#define LOG(x) std::cout << x
+#define DLOG(x) std::cout << x
```

Comment:
missing new line. Also license header is missing. Please take a look at the other .h files.